py/migrate.py

```diff
@@ -12,46 +12,42 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Slurp the local db/ directory contents into Postgres."""
+"""Slurp local datasets into Postgres."""
 
-import os
+import glob
 import re
-import sys
-import time
 
+from absl import app
+from absl import flags
 import psycopg2
 import psycopg2.sql
 
+FLAGS = flags.FLAGS
+flags.DEFINE_string('input_pattern', None, 'Input dataset glob.')
 
-def migrate_one(user_id, name, conn):
-    """Slurp one named dataset from the db/ directory into Postgres."""
-    pbtxt = open(f'db/{user_id}/{name}').read()
-    query = psycopg2.sql.SQL(
-        'INSERT INTO datasets VALUES (%s, %s, %s) '
-        'ON CONFLICT (user_id, dataset_name) DO UPDATE SET pbtxt=%s')
```

Comment:
I recommend keeping the "ON CONFLICT ... DO ...". It's good for the script to be idempotent, so we can iterate running it.

Reply:
The UPDATE automatically makes sure we can iterate...? ON CONFLICT seems to be more useful for INSERT